content/articles/how-to-ground-truth-label-your-datasets-for-creating-autonomous-driving-systems-using-matlab-ground-truth-app/index.md

```diff
@@ -0,0 +1,180 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /how-to-ground-truth-label-your-datasets-for-creating-autonomous-driving-systems-using-matlab-ground-truth-app/
```

Comment:
The URL needs to be short and precise. Please change the name of the folder. Thank you.